framework/src/modules/http_api/controllers/node.js

```diff
@@ -124,7 +121,7 @@ async function _getConfirmedTransactionCount() {
 			);
 		} catch (error) {
 			// Ignore error and just put warn
-			library.components.logger.warn("Transaction count wasn't cached", error);
+			library.components.logger.debug(error, "Transaction count wasn't cached");
```

Comment:
this is invalid log message, it should be `Failed to set transaction count` or something like that

Reply:
I think it is correct as meaning, but since it's the same error message so ill update it